release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -289,6 +289,11 @@ The cluster network can be expanded to support the addition of nodes to the clus
 
 In {product-title} {product-version}, {lvms} is supported in dual-stack for IPv4 and IPv6 network environments. For more information, see xref:../networking/ovn_kubernetes_network_provider/converting-to-dual-stack.adoc#nw-dual-stack-convert_converting-to-dual-stack[Converting to a dual-stack cluster network].
 
+[id="ocp-4-13-storage-lvms-in-zpt"]
+==== Support for {lvms} in GitOps ZTP
+
+In {product-title} {product-version}, you can add and configure {lvms-first} through GitOps ZTP. For more information, see xref:../scalability_and_performance/ztp_far_edge/ztp-manual-install.adoc#ztp-lvms-installing-using-web-console_ztp-manual-install[Intalling LVM Storage by using the web console], xref:../scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.adoc#ztp-provisioning-lvm-storage_ztp-advanced-policy-config[Configuring LVM Storage using PolicyGenTemplate CRs], and xref:../scalability_and_performance/ztp_far_edge/ztp-reference-cluster-configuration-for-vdu.adoc#lvms-configuring-lvms-on-sno_sno-configure-for-vdu[LVM Storage].
```

Comment:
Two of the links don't seem to match the link text that you mention.

- The first link, `xref:../scalability_and_performance/ztp_far_edge/ztp-manual-install.adoc#ztp-lvms-installing-using-web-console_ztp-manual-install`, opens a page entitled "Manually installing a single-node OpenShift cluster with ZTP".

- The third link, `xref:../scalability_and_performance/ztp_far_edge/ztp-reference-cluster-configuration-for-vdu.adoc#lvms-configuring-lvms-on-sno_sno-configure-for-vdu`, opens a page entitled "Recommended single-node OpenShift cluster configuration for vDU application workloads".

Reply:
Now, links are working as intended